watch-parties/src/main/webapp/index.html

```diff
@@ -1,16 +1,32 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <script src="script.js"></script>
-        <link rel="stylesheet" href="style.css">
-    </head>
-    <body onload="loadVideoInfo()">
-        <div id="player"></div>
+
+  <head>
+    <meta charset="UTF-8">
+    <title>Watch Parties</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="chatstyle.css">
+    <script type="module" src="script.js"></script>
+    <script type="module" src="chatscript.js"></script>
+  </head>
+  <body onload="loadVideoInfo()">
+      <div id="player"></div>
+
 
         <div id="video-info">
 
             <h2 id="video-title"></h2>
 
         </div>
-    </body>
+
+      <div id=“chat” style="float:right;">
+        <ul id="chatlog">
+            <br />
+        </ul>
+        <ul id="new-comment">
+            <li><textarea id="submitted-comment" style="background-color:#faf2f0;width:336px;"></textarea>
+            <button style="background-color:#faf2f0;border-width:1px;" id="sendButton" method="GET" action="chatmessages"><img src="images/sendicon.png" style="width:40px;height:40px;padding:0px;"></button></li>
```

Comment:
The `method` and `action` attributes aren't supported on `<button>` (those are only for `<form>`) so they can be removed.